app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/subscription/do_you_have_a_trn.scala.html

```diff
@@ -49,7 +49,7 @@
             errorKey = form.error(key).map(e => e.message),
             hasErrors = hasErrors,
             selected = form.value,
-            helpText = Some(Html(messages("haveATrn.helpText", appConfig.trusteeResponsibilitiesUrl)))
```

Comment:
is it likely we'll every use these urls again? If not, we could clean up `application.conf` and remove them from there

Reply:
The only one we can remove is ninoGuidanceUrl - the others are still used elsewhere